rust/processor/src/processors/account_transactions_processor.rs

```diff
@@ -101,20 +102,24 @@ impl ProcessorTrait for AccountTransactionsProcessor {
         let processing_start = std::time::Instant::now();
         let last_transaction_timestamp = transactions.last().unwrap().timestamp.clone();
 
-        let mut account_transactions = AHashMap::new();
-
-        for txn in &transactions {
-            account_transactions.extend(AccountTransaction::from_transaction(txn));
-        }
-        let mut account_transactions = account_transactions
-            .into_values()
-            .collect::<Vec<AccountTransaction>>();
-
-        // Sort by PK
-        account_transactions.sort_by(|a, b| {
-            (&a.transaction_version, &a.account_address)
-                .cmp(&(&b.transaction_version, &b.account_address))
-        });
+        let account_transactions: Vec<_> = transactions
+            .into_par_iter()
+            .enumerate()
+            .map(|(i, txn)| {
+                let transaction_version = start_version as i64 + i as i64;
```

Comment:
We should get the version from `txn` instead of doing addition here which might be wrong.